packages/client/src/runtime/core/engines/library/LibraryEngine.ts

```diff
@@ -123,6 +132,26 @@ export class LibraryEngine implements Engine<undefined> {
     }
   }
 
+  async applyPendingMigrations(): Promise<void> {
+    if (TARGET_BUILD_TYPE === 'rn') {
+      await this.start()
+      // @ts-ignore
```

Comment:
Please, create global `__PrismaProxy` definition rather than `@ts-ignoring` every usage.

Reply:
The __PrismaProxy is defined in `react-native-prisma`. I can create a type, but that would mean they both need to be kept in Sync whenever the C/Rist bindings change. I can do it, just want to confirm you want to take this route.